framelesswindow/framelesswindow.cpp

```diff
@@ -164,13 +169,21 @@ void FramelessWindow::on_minimizeButton_clicked()
 }
 
 void FramelessWindow::on_restoreButton_clicked() {
+  if (  QSysInfo::productType() == "windows"

```

Comment:
this condition is obsolete - it works also for Mac and Linux and Win 10
so you can remove it again.
Btw. you can just make changes and commit on same branch - if you push it, the commit will appear in this merge request. So you do not need to create a new one.